_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -1740,7 +1740,12 @@
 /obj/machinery/camera/directional/south{
 	c_tag = "Hallway - Western Tram Bridge 2"
 	},
-/turf/open/floor/vault,
+/turf/open/floor/vault{
+	base_icon_state = "tram2";
+	icon_state = "tram2";
+	desc = "A sturdy looking tram platform.";
+	name = "tram platform"
+	},
 /area/station/hallway/primary/tram/center)
```

Comment:
why not a subtype?

Reply:
Yeah, we could do that. No subtype as it exists, but can improve on it.